src/mse/virtual-source-buffer.js

```diff
@@ -491,6 +490,9 @@ export default class VirtualSourceBuffer extends videojs.EventTarget {
       triggerUpdateend = true;
     }
 
+	//Add text-track data for all
+	addTextTrackData(this, sortedSegments.captions, sortedSegments.metadata);
```

Comment:
Looks like the build is failing since this should be indented 2 more spaces (and the comment too)